### ISSUE TITLE
App rebranding

### ISSUE THREAD

**Issue by Tunous** [Tunous/Dank#8]
A lot of people didn't like the name *Dank* for the app. Perhaps it's a good time to change the name to something which would sound more professional? I personally very like the name **Seek** which someone shared in dank's subreddit previously but I'm open to other ideas.

@saket are you ok with this, or maybe you have other suggestions?
Also while I have a chance do you have any idea how we could work out to make the app available on the play store? I technically could create my own signing keys and upload them under my account  (package name would change with rebranding to avoid conflict) but it kind of feels weird to have my name as the developer since you've made almost all of the work with it. (I can contact you somewhere else if you prefer)

**Comment by Tunous** [comment]
> Let me think about this. Re-branding the original repository will be better so that all forks receive the updates.

Yes, that would be the best option.

> I have started considering pulling your changes upstream. It makes me so happy to see you all making constant bug fixes and feature additions.

That sounds great! I hope that you'll decide to do it 😀

> Let me see if I can afford doing this actively. If it works out, I'll keep uploading the updates. In the meanwhile, could you please upload the releases on Github?

Definitely, my aim was to make as much as possible work automatically. One of the things I did is setup CircleCI (https://circleci.com/gh/Tunous/Dank). The config file I created includes jobs for building debug version of the app and also a job for uploading them to GitHub releases on each tag push to repository. It already works correctly in my fork. The same thing can be done with release builds.
It's also possible to automatically publish them to alpha channel on Play Store with help of Gradle plugins.

With all of that it would be possible to create a release by just pushing a tag to repository and it would land in Google Play alpha channel (or other depending on config)

**Comment by voythas** [comment]
About the name, perhaps new name should be cosmos/astronomy oriented? I do like the name Apollo for the iOS client and Snoo is an alien so perhaps Dank should also follow that lead. Good I did not start working on new app icon then.

**Comment by saket** [comment]
> Definitely, my aim was to make as much as possible work automatically. One of the things I did is setup CircleCI (https://circleci.com/gh/Tunous/Dank). The config file I created includes jobs for building debug version of the app and also a job for uploading them to GitHub releases on each tag push to repository. 

This looks good. I'll import it when possible.

**Comment by saket** [comment]
Alright let's do this: https://github.com/saket/Dank/pull/6

**Comment by Esselans** [comment]
> A lot of people didn't like the name _Dank_ for the app. Perhaps it's a good time to change the name to something which would sound more professional? I personally very like the name **Seek** which someone shared in dank's subreddit previously but I'm open to other ideas.

I like **Seek**. I tried thinking a name with 'swype' or 'gestures' but nothing came up.

**Comment by lastweakness** [comment]
Seek sounds good. BTW, any place to download debug builds?

**Comment by PoorPocketsMcNewHold** [comment]
> > A lot of people didn't like the name _Dank_ for the app. Perhaps it's a good time to change the name to something which would sound more professional? I personally very like the name **Seek** which someone shared in dank's subreddit previously but I'm open to other ideas.
> 
> I like **Seek**. I tried thinking a name with 'swype' or 'gestures' but nothing came up.

Around the same idea, I've tough of a rime with Reddit, Like "SlickIt" or with yours "SeekIt". _Yeah, I'm not that good with English Slang_
However, Outside of that, The Dank icon need also to be changed, as, Deal With It glasses are "2 Dank 4 Me" still.
So, I'll suggest instead another type of glasses, with an overhaul more classical look. I did conserve the swipe upvotes and downvotes off the app on the icon however to give it more identity. 
That's just a suggestion, I'm not a professional or whatever. 
![SeekIt/SlickIt](https://framapic.org/vEmpa7TkOupN/x4xTEjYbNxUT)

**Comment by az4726** [comment]
@PoorPocketsMcNewHold That icon looks very good, but I would suggest making the icon adaptive so that it fits the user's preferred icon theme. So the top layer could be the reddit icon with the upvotes, and the back layer could be the purple backdrop or maybe even the shade of grey used in the app theme. But that looks really good!

**Comment by dotvhs** [comment]
@PoorPocketsMcNewHold Icon looks fine but I would remove the up/down vote and leave the head only? For me it's a bit too noisy right now. But it's a good start.

**Comment by Esselans** [comment]
> @PoorPocketsMcNewHold Icon looks fine but I would remove the up/down vote and leave the head only? For me it's a bit too noisy right now. But it's a good start.

Maybe put the upvote on one side of the glasses and the downvote on the other side (on the inside).

**Comment by PoorPocketsMcNewHold** [comment]
> @PoorPocketsMcNewHold That icon looks very good, but I would suggest making the icon adaptive so that it fits the user's preferred icon theme. So the top layer could be the reddit icon with the upvotes, and the back layer could be the purple backdrop or maybe even the shade of grey used in the app theme. Once again that looks great!
> 
> [https://developer.android.com/guide/practices/ui_guidelines/icon_design_adaptive](url) Heres a link to the android developer docs for a better explanation of what I meant

![dankiconadaptiveblacknoshadow](https://user-images.githubusercontent.com/13170204/51250111-bccec600-1995-11e9-99ee-e9d82d2c6375.png)
![dankiconadaptive](https://user-images.githubusercontent.com/13170204/51250112-bccec600-1995-11e9-80f5-1251bd88f736.png)
Am i doing it right ?


> @PoorPocketsMcNewHold Icon looks fine but I would remove the up/down vote and leave the head only? For me it's a bit too noisy right now. But it's a good start.

![dankiconmockupwithoutupvotes](https://user-images.githubusercontent.com/13170204/51250163-dbcd5800-1995-11e9-8952-fa0751e67a8f.png)

I suppose, like that ?
> > @PoorPocketsMcNewHold Icon looks fine but I would remove the up/down vote and leave the head only? For me it's a bit too noisy right now. But it's a good start.
> 
> Maybe put the upvote on one side of the glasses and the downvote on the other side (on the inside).

![dankiconfromsidetoside](https://user-images.githubusercontent.com/13170204/51250202-ee479180-1995-11e9-9f81-d40319b30bd3.png)
![dankiconsidetoside](https://user-images.githubusercontent.com/13170204/51250204-eee02800-1995-11e9-8695-cdbd7870630d.png)
Which one were you trying to describe ? (Except, if it was a mix of both at different line starting points)

**Comment by Esselans** [comment]
@PoorPocketsMcNewHold don't laugh about my paint skills but I meant something like this:

![2019-01-16 16_30_03-window](https://user-images.githubusercontent.com/12036503/51273774-45f0f780-19ac-11e9-890d-96be2d93556c.png)

But it's a terrible idea xD

**Comment by dotvhs** [comment]
Snoo-only icon looks best if you ask me. I will give it a go one day as well, I have few ideas.

**Comment by dotvhs** [comment]
I am not really sure about the colors just yet, so please bear with me:
![image](https://user-images.githubusercontent.com/681416/51433340-7e146680-1c48-11e9-94e9-e27b73105ada.png)

**Comment by dotvhs** [comment]
Small revisions (for now I left the antenna as is):
![microsoft](https://user-images.githubusercontent.com/681416/51435118-710b6d80-1c70-11e9-9615-ab36b4deddd1.png) 
![microsoft](https://user-images.githubusercontent.com/681416/51435122-9009ff80-1c70-11e9-8762-f2937ea6a6d9.png)
![microsoft](https://user-images.githubusercontent.com/681416/51435134-c182cb00-1c70-11e9-9bbb-50c81d8a2d56.png)

**Comment by PoorPocketsMcNewHold** [comment]
> Small revisions:
> ![microsoft](https://user-images.githubusercontent.com/681416/51435480-5b01ab00-1c78-11e9-98d7-e4df9c6fc87e.png) ![microsoft](https://user-images.githubusercontent.com/681416/51435492-8a181c80-1c78-11e9-838d-62a6996ad549.png)
> 
> I still need to work on the shape of that antenna, there is something off about it.

I absolutely love these. I would even recommend using the "Vice City" colored one as well as for a dev/beta branch one, and the other one for the stable branch.

**Comment by qtto** [comment]
@Esselans Good question, and sadly, reddit generally doesn't allow snoo to be used in app logo's. See https://www.redditinc.com/assets/press-resources/reddit_brand_guidelines_20180314.pdf, page 16:

> **Generally not permitted or approved**
> Use of the Reddit brand as any part of the name of a company, product, service, app, website, domain name, or social media account.

So sadly, these logo ideas might get the app pulled from the Play Store/App Store.

**Comment by dotvhs** [comment]
@qtto Well, dang it. Let's hope that we are fine as long as we don't use original snoo, which has different colours, eyes, a smile, antenna and an outline.

**Comment by qtto** [comment]
Let's hope so indeed - if reddit thinks it's too similar they'll have you change it (this happened with Apollo). I do think this might be fine.

I dig the icons @Esselans proposed, but I'm not sure if it should stay with the original theme? If the app is rebranded to Seek I think the icon should not be stuck in the Dank-era (i.e. the sunglasses).

**Comment by Soitora** [comment]
Has there been any progress overall on the rebrand?

Really beautiful designs @dotvhs

**Comment by Tunous** [comment]
I was about to publish the app to the play store but then having to upload an icon reminded me about this discussion. Now, I guess it would be a good moment to go for the new name (or keep dank) and icon design.

@saket what do you think about this? You mentioned doing it in main repository too so I wonder how we could pull that out.

**Comment by PoorPocketsMcNewHold** [comment]
Well, I think we should profit this occasion of renaming Dank. 
Compared to the currently popular Reddit clients, Dank has short-lived and didn’t receive as much love as I would think, to keep the need to re-use that name for its rebrand.
We can indeed relaunch the name brainstorming (But the already proposed names for it were pretty good already)

**Comment by saket** [comment]
Um, go ahead with changing the name for your fork. I think I'm okay with forks maintaining their own names and logos. Just keep a credit link somewhere in the about section and I'll be more than happy :)

**Comment by qtto** [comment]
I agree that changing the name would be good. When the app was first released by @saket it already had a lot of comments about the name :-)

I think Seek was a fine suggestion - it makes it sound a lot like a video/music player in my opinion, but at least it implies some movement (good for such a gesture focused app). But I do believe brainstorming some more would be nice.

**Comment by Esselans** [comment]
Yeah. I'll go with *Seek* and the logos made by @dotvhs (the left one)

**Comment by Soitora** [comment]
**Seek for Reddit** with the app icons from @dotvhs

**Comment by dotvhs** [comment]
If it's needed, I can work on some more designs but it seems you guys liked last version.

**Comment by Tunous** [comment]
I would be ok with using the yellow version for regular builds and purple for dev but I'm not really sure whether we should risk having the app pulled from store because of it. Right now it looks too close to original snoo since we can see the whole shape of head.

@dotvhs I might leave the decision up to you. Are you willing to send us required assets to the latest version or do you want to have another attempt with something different? **Seek** is a good name for this fork, perhaps sunglasses won't fit to that as someone said?

**Comment by PoorPocketsMcNewHold** [comment]
It is possible to contact Reddit directly to ask them suggestion on the use of it ?

-------- Message d'origine --------

**Comment by Esselans** [comment]
https://www.reddit.com/wiki/licensing

**Comment by Tunous** [comment]
![jGZr88Kx3j](https://user-images.githubusercontent.com/5156340/63608306-ece9e180-c5d4-11e9-8eb1-5fbd1a71b5c8.gif)

This is how it looks with what we've got so far.

**Comment by dotvhs** [comment]
> I would be ok with using the yellow version for regular builds and purple for dev but I'm not really sure whether we should risk having the app pulled from store because of it. Right now it looks too close to original snoo since we can see the whole shape of head.

I don't think we need to worry, about it, original Snoo has different antenna and no sunglasses, so while it resembles the Snoo, it's not using their assets and if we use version with dark blue background and green head - I think we will be safe for sure, since Snoo is white.

> @dotvhs I might leave the decision up to you. Are you willing to send us required assets to the latest version or do you want to have another attempt with something different? **Seek** is a good name for this fork, perhaps sunglasses won't fit to that as someone said?

Of course I'll send prepared assets, I can work on other revisions later.

**Comment by Tunous** [comment]
Also we'll need 24dp full white version for notifications, preferably in xml :)

Current looks like this:
![image](https://user-images.githubusercontent.com/5156340/63609279-215e9d00-c5d7-11e9-8a4c-e8bf25496180.png)

**Comment by qtto** [comment]
Definitely not to be rude to the creator, because these icons are designed very well - but I think the icon might not be suitable in the long run. Not only because Reddit might complain (a different look for snoo is not necessarily enough for them to excuse it: see slide or relay's old icons) but also because it doesn't quite fit the name, in my opinion. Seek is a good name, but the icon doesn't really have a relation to it.

However, I don't have a better icon right now, and I think changing the icon later shouldn't be much of a problem :-)

**Comment by dotvhs** [comment]
> Also we'll need 24dp full white version for notifications, preferably in xml :)
> 
> Current looks like this:
> ![image](https://user-images.githubusercontent.com/5156340/63609279-215e9d00-c5d7-11e9-8a4c-e8bf25496180.png)

Okay, I'll prepare it too.


> Definitely not to be rude to the creator, because these icons are designed very well - but I think the icon might not be suitable in the long run. Not only because Reddit might complain (a different look for snoo is not necessarily enough for them to excuse it: see slide or relay's old icons) but also because it doesn't quite fit the name, in my opinion. Seek is a good name, but the icon doesn't really have a relation to it.
> 
> However, I don't have a better icon right now, and I think changing the icon later shouldn't be much of a problem :-)

Agreed, maybe we should change the name then? Vice for Reddit :D or Funk for Reddit, New Wave, Retro Wave for Reddit, Wave for Reddit? Dawn for Reddit?  I dunno. I'm neutral to the Seek name.

Or maybe we should make a new logo after all, but I currently don't have any fancy ideas for it, so I'm open.

**Comment by Esselans** [comment]
I only came up with Glide For Reddit, or only Glide. And of logo use something like that. I am aware of the Glide library that is use on this project but It also applies to things like smooth.

![256x256bb](https://user-images.githubusercontent.com/12036503/63627311-8b7a4080-c5dd-11e9-99c8-4ad48fb21aeb.jpg) 

But with a (similar to) Snoo.

**Comment by Tunous** [comment]
> Dawn for Reddit

I like that name too. Could use that. Alternatively the opposite: **Dusk for Reddit**

**Comment by qtto** [comment]
I also really like Dawn! Dusk as well, but dawn of course signifies a "new day", or rather, something like new content in our case. Fits well!

**Comment by PoorPocketsMcNewHold** [comment]
I personally prefer largely the Dawn for reddit aswell. Especially since it reflect more the current icon aswell.

**Comment by Soitora** [comment]
I can actually get behind the name **Dusk for Reddit**, "similiar" to dank and more closely fits with the logo as well as the activity (who else reads Reddit until late night?)

In that case, the Dev icon/loader in this should be used for Release instead as it fits better with the name.

> | Release (Should be Dev) | Dev (Should be Release) |
> | --- | --- |
> | ![jGZr88Kx3j](https://user-images.githubusercontent.com/5156340/63608306-ece9e180-c5d4-11e9-8eb1-5fbd1a71b5c8.gif) | ![eLK8XIAW5p](https://user-images.githubusercontent.com/5156340/63609086-b319da80-c5d6-11e9-9e21-f7a27eccf314.gif) |

**Comment by Tunous** [comment]
Let's go for **Dawn for Reddit**

**Comment by Soitora** [comment]
Sounds good, **Dawn for Reddit** with the Dawn-logo, and Dusk-logo for Dev release

**Comment by Tunous** [comment]
@dotvhs Any progress on the designs, did you manage to find time to export them for app and store?

**Comment by elenmerbau** [comment]
> Also we'll need 24dp full white version for notifications, preferably in xml :)
Maybe use the sun thingy? Something like this one:
![Untitled-1](https://user-images.githubusercontent.com/23418072/68859817-50863580-0712-11ea-9d35-aabacac179f8.png)

**Comment by PoorPocketsMcNewHold** [comment]
I found it great, but i'm scared that it might not be recognizable. I would have suggested the frame of the glasses alone, which are directly recognizable.

**Comment by elenmerbau** [comment]
> I found it great, but i'm scared that it might not be recognizable. 

Yeah, I had the same thoughts.

> I would have suggested the frame of the glasses alone, which are directly recognizable.

Might look like a headset :p

**Comment by dotvhs** [comment]
> @dotvhs Any progress on the designs, did you manage to find time to export them for app and store?

Sorry for lack of answer, real life sometimes hits hard. Expect those icons this weekend. I will also prepare notification icons too in few version and I'll let you guys pick :)

**Comment by Tunous** [comment]
@dotvhs Hello, do you have any new information about these designs for us?

**Comment by shore-leave** [comment]
Because it has been a while since @dotvhs (hope you're ok, also hope it's ok to modify your work like this) checked in I tried to continue the work they had done on the icons.

Changes I made:
- Antenna has a new angle
- Sunglasses are slightly steeper
- Flipped the sunglasses gradient direction for Dusk (not sure if it's better, would like feedback)
- Added a small slope to the sunglasses gradient steps
- Added a slight gradient to the gradient steps for the Dusk icon (see palette)
- Added a small slope to the background cutouts
- Switched the background to a non-stepped gradient
- Added slight drop-shadow to the head
- Added chromatic drop shadow to sunglasses and antenna ball

All in all, would like feedback. Can provide source (.afdesign or rendered .svg)

|                   | Dusk | Dawn |
|:------------------|:----:|:----:|
| App Icon          | ![tunous_dawn_dusk_icon](https://user-images.githubusercontent.com/65641441/82404089-58f75a00-9a2e-11ea-9ef7-f11c83eb8056.png) | ![tunous_dawn_dawn_icon](https://user-images.githubusercontent.com/65641441/82404087-58f75a00-9a2e-11ea-8640-54ac8f12d7ff.png) |
| Notification Icon | ![tunous_dawn_dusk_notification](https://user-images.githubusercontent.com/65641441/82404086-585ec380-9a2e-11ea-9673-3bee69c75b5c.png) | ![tunous_dawn_dawn_notification](https://user-images.githubusercontent.com/65641441/82404084-585ec380-9a2e-11ea-95f2-f8019b4bc4ed.png) |
| Color Palette     | ![tunous_dawn_dusk_palette](https://user-images.githubusercontent.com/65641441/82404092-598ff080-9a2e-11ea-9d57-d3853fbd44ee.png) | ![tunous_dawn_dawn_palette](https://user-images.githubusercontent.com/65641441/82404091-598ff080-9a2e-11ea-88d9-f27e06c46bdc.png) |

**Comment by Tunous** [comment]
@shore-leave Great work!

There are few things that we need to implement this correctly.

First of all for the Play Store icon we'll need 512x512 png image. There is a link with exact details about what is needed: https://developer.android.com/google-play/resources/icon-design-specifications#attributes (I can see that there are also templates at the bottom of he page but have no idea if these are of any use)

Next for the adaptive icon we'll need two layers. If possible the best option would be to get both of them exported to xml format understood by Android. If not then two png images in 512x512 should be good too. https://developer.android.com/guide/practices/ui_guidelines/icon_design_adaptive here is some more information about this.

For splash screen I’m not sure, perhaps the same images as for adaptive icon will work.

As for the notification icon, svg will be good :)

**Comment by shore-leave** [comment]
#212

**Comment by shore-leave** [comment]
Putting this out here as a possible direction to take the app UI. Feedback would be appreciated. The color scheme will be replacing the current one to match the rest of the new app branding.

![tunous_dawn_app_04](https://user-images.githubusercontent.com/65641441/84555000-a3e05680-ace8-11ea-98de-0a5979e1c453.png)

**Comment by saket** [comment]
That top bar looks very nice to me!

**Comment by Tunous** [comment]
@shore-leave That's an interesting concept, especially with the color scheme and the top bar. Few thoughts from me:

1. What is the meaning of icon on the right of "popular"?
2. The toolbar is missing refresh button, going for pull to refresh is not really an option as that gesture is reserved for closing.
3. I see that you turned the appearance of submissions upside down with subreddit info/points at the top and title at the bottom. I really like how the score appears next to title in current version and wouldn't want to have this changed. But perhaps we just need to see a design with all current features included to compare this correctly. (images, author/comments info, status text)

**Comment by elenmerbau** [comment]
That FAB button is quite big for users who don't post much.

**Comment by shore-leave** [comment]
@saket Thanks. Still more work to be done. At least there might be something there

@Tunous
1. The icon to the right of popular is the (un)subscibe button. When it's checked it means that this subreddit is in your list of subscriptions, and when there's a plus it means that you can click it to subscribe. If that isn't obvious enough there's room for exploration for a more discoverable version
2. I forgot to put that in. The top bar feels heavy to the sides, so I was considering moving those top right actions to an overflow menu. That might also help with the transition when opening the account dialog. Right now the expanding sheet we use would need to work around the items below it on the bar. I noticed that the only 3-dot overflow menus in the app are opened by swipe gestures. Is that a requirement?
3. That is fine. I have no preference to how the score shows, except that I would like it to not push the submission titles different amounts. Maybe a set reserved minimum space for it to cover 3 digits being displayed, which it can expand past for the rare posts with over 100k?

My focus overall is to get the color scheme changed, and explore a small part of improving UX. If the colors look good I can start implementing that

**Comment by shore-leave** [comment]
@elenmerbau Noted! I think the standard is for those to shrink down to a regular FAB on scroll down

Speaking about scrolling down: I didn't mention this earlier, but the top row of the top bar would scroll out of view when you scroll down. The bottom actions that relate to the subreddit you are in would stay on screen

**Comment by saket** [comment]
The notification icon is barely visible 

[!icon](https://i.imgur.com/bwp59ex.jpg)

**Comment by shore-leave** [comment]
Thanks for pointing that out @saket. I was worried it would look that way. What screen resolution is the device you are using? From looking at it I would guess that I need to scale up the white parts by 2x.

I always have trouble guessing how it will look at small sizes, so it would be nice to get a way to test them in the app. Maybe someone can implement that. I could also build a template in Affinity Designer to help test how they look at different resolutions.

**Comment by fushinari** [comment]
@shore-leave Isn't it okay for both Dusk and Dawn to use the same notification icon? Dawn's notification icon is great from a design perspective, but not visible enough on smaller displays. Maybe just use Dusk's icon instead?
